evil-collection.el

```diff
@@ -34,7 +34,11 @@
 ;;      (evil-minibuffer-init)
 
 ;;; Code:
-(defvar evil-collection-mode-list
+(defgroup evil-collection nil
+  "A set of keybindings for Evil mode"
+  :group 'evil)
```

Comment:
Group is probably 'evil-collection or 'evilcoll (after I make the namespace changes).